src/Microsoft.Identity.Web.TokenAcquisition/MsalAspNetCoreHttpClientFactory.cs

```diff
@@ -17,7 +17,9 @@ public MsalAspNetCoreHttpClientFactory(IHttpClientFactory httpClientFactory)
 
         public HttpClient GetHttpClient()
         {
-            HttpClient httpClient = _httpClientFactory.CreateClient();
+            // The HttpClient will be created with the named client registered in the DI container
+            // If the named client is not found, the factory will fall back to the default client
+            HttpClient httpClient = _httpClientFactory.CreateClient("token_acquisition_httpclient");
```

Comment:
I'm confused. 

I thought we had agreed that in the constructor of the `TokenAcquisition` class, you would no longer new-up and `MsalAspNetCoreHttpClientFactory(),` but you would use the locator pattern (that is attempt to get a service implementing `IMsalHttpClientFactory` from the service provider):

```CSharp
 _httpClientFactory = serviceProvider.GetService<IMsalHttpClientFactory>() ??  new MsalAspNetCoreHttpClientFactory(httpClientFactory);
```

did the design change @victorm-hernandez @ciaozhang ?

Reply:
@jmprieur , as per our last conversation we agreed to use a named HTTP client. I will share the diagram we reviewed last time over email.